doc/whats_new/v0.20.rst

```diff
@@ -86,6 +86,16 @@ Changelog
 - |Fix| Fixed bug in :class:`preprocessing.OrdinalEncoder` when passing
   manually specified categories. :issue:`12365` by `Joris Van den Bossche`_.
 
+:mod:`sklearn.utils`
+........................
+
+- |Fix| Use float64 for mean accumulator in
+  :func:`utils.extmath._incremental_mean_and_var` to avoid floating point
+  precision issues when using float32 datasets. Affects
+  :class:`preprocessing.StandardScaler` and
+  :class:`decomposition.IncrementalPCA`.
+  :issue:`12333` by :user:`bauks <bauks>`.
```

Comment:
Actually this should reference the PR and we can't mention a private function in the what's new, only the effect of this change on public estimators. I'll fix it ..